src/main/java/reactor/netty/resources/ConnectionProvider.java

```diff
@@ -274,6 +275,19 @@ public final Builder maxLifeTime(Duration maxLifeTime) {
 			return this;
 		}
 
+		/**
+		 * Switch order for the channels in the pool to LIFO.
+		 * <br>
+		 * Default FIFO.
+		 *
+		 * @param lifo flag for switching order for the channel to LIFO
+		 * @return {@literal this}
+		 */
+		public final Builder lifo(boolean lifo) {
```

Comment:
Let's change this method to two methods -> `fifo()` and `lifo()`

Reply:
@violetagg ,what do think about this approach? should i use primitive or Object boolean?
i mean, if we'll use primitive boolean then we don't need to use (e.g.) fifo(), because it will use by default.
or do we need to explicitly use one of the methods?

Reply:
The point is that at the moment we have only two types of leasing strategies, so using the approach that you implemented is OK for the moment. But if we decide to add a third type of leasing strategy we will not be able to do it.

Reply:
aaa, ok, thanks :)